src/pages/AssignServerRoles.js

```diff
@@ -53,7 +53,7 @@ class AssignServerRoles extends BaseWizardPage {
   constructor(props) {
     super(props);
 
-    this.IS_INSTALLL = this.props.mode ? false : true;
+    this.IS_INSTALLL = this.props.mode === undefined ? false : true;
```

Comment:
just noticed, too many LLL's here like was previously the case in EditServerDetails.js

Reply:
also, isnt this the reverse of the intended logic? if this.props.mode is undefined, IS_INSTALL should be true?